database/dbConfig.js

```diff
@@ -0,0 +1,7 @@
+const knex = require('knex');
+
+const knexConfig = require('../knexfile');
+
+const environment = process.env.DB_ENV || 'development'
```

Comment:
No need for the OR if we add DB_ENV to our .env-s!

Reply:
good point

Reply:
It's better to have it for foolproofing once again

export is missing though